src/frontend/components/SøknadsSteg/OmDeg/OmDeg.tsx

```diff
@@ -35,6 +41,17 @@ const OmDeg: React.FC = () => {
         utenlandsperioder,
     } = useOmdeg();
 
+    let leggTilPeriodeTekster: ReturnType<typeof hentLeggTilPeriodeTekster> = undefined;
```

Comment:
Du kan skrive denne en del enklere. Eksempelvis:
```
const antallPerioder = utenlandsperioder.length;

const leggTilPeriodeTekster = toggles.NYE_MODAL_TEKSTER && hentLeggTilPeriodeTekster(
            'utenlandsopphold',
            PersonType.Søker,
            antallPerioder
        );
```

`&&` gjør slik at den andre verdien blir satt på const'en kun når logikken for toggles returnerer true. 😊

Reply:
Når jeg tenker meg om tror jeg det kan være enklere å legge toggle-logikken inn i `LeggTilKnapp`. Hva tenker du? Så slipper du å hver gang sjekke om du skal sende med forklaringen eller ikke. :)